app/src/main/java/ahhhlvin/c4q/nyc/vinecodingproject/IssuesActivity.java

```diff
@@ -1,158 +0,0 @@
-package ahhhlvin.c4q.nyc.vinecodingproject;
-
-import android.os.AsyncTask;
-import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.LinearLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.support.v7.widget.Toolbar;
-import android.util.Log;
-import android.view.Menu;
-import android.view.MenuItem;
-
-import org.json.JSONArray;
-import org.json.JSONException;
-import org.json.JSONObject;
-
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.util.ArrayList;
-
-public class IssuesActivity extends AppCompatActivity {
-
-    private static final String REQUEST_METHOD_GET = "GET";
-    private static final String ENDPOINT = "https://api.github.com/repos/rails/rails/issues";
-    private ArrayList<GitIssue> mIssuesList;
```

Comment:
change this to List<GitIssue> idk why but make it more abstract is better, I heard?